app/upgrade_handler.go

```diff
@@ -6,47 +6,32 @@ import (
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/cosmos/cosmos-sdk/x/authz"
-	sdkauthz "github.com/cosmos/cosmos-sdk/x/authz"
-	"github.com/cosmos/cosmos-sdk/x/feegrant"
-	sdkfeegrant "github.com/cosmos/cosmos-sdk/x/feegrant"
+	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-
-	ibcconnectiontypes "github.com/cosmos/ibc-go/v2/modules/core/03-connection/types"
-
-	shieldtypes "github.com/certikfoundation/shentu/v2/x/shield/types"
 )
 
 const (
-	v230Upgrade = "Shentu-v230"
-	shieldv2    = "Shield-V2"
+	tmp = "tmp"
 )
 
-func (app ShentuApp) setv230UpgradeHandler() {
+// TODO: rename upgrade title
+func (app ShentuApp) setTmpUpgradeHandler() {
 	app.UpgradeKeeper.SetUpgradeHandler(
-		v230Upgrade,
-		func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
-			app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
-
-			fromVM := make(map[string]uint64)
-			for moduleName := range app.mm.Modules {
-				fromVM[moduleName] = 1
+		tmp,
+		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			migrationOrder := make([]string, len(fromVM))
```

Comment:
Correct me if I'm wrong，fromVM doesn't include crisis module because it's new, right?

Reply:
We want it last since shield module has a broken invariant due to a bug that messed some parts of the bookkeeping of the coins. so if the initgenesis of crisis module runs before shield store migration (which should fix that broken invariant) module it'll panic